File: cpp_e9zMZ/cpp/02-caudieukien/50.cpp
#include <iostream>
using namespace std;

int main() {
    int yourNumber = 0;
    string strName = "dương";
    string strEven = "chẵn";

    if (yourNumber < 0) {
        strName = "âm";
    }

    if (yourNumber % 2 != 0) {
        strEven = "lẻ";
    }

    cout << "Số nguyên " << strName << " " << strEven;
    return 0;
}

File: python_ToSXs/python/02-caudieukien/50.py
yourNumber = 4

strOne = "dương"
strTwo = "chẵn"

if yourNumber < 0:
    strOne = "âm"
if yourNumber % 2 != 0:
    strTwo = "lẻ"

print(f"Số nguyên {strOne} {strTwo}")
